guarddog/analyzer/sourcecode/npm-silent-process-execution.yml

```diff
@@ -0,0 +1,46 @@
+rules:
```

Comment:
inspired from https://github.com/returntocorp/semgrep-rules/blob/develop/javascript/lang/security/audit/dangerous-spawn-shell.yaml but I agree the rule looks complex overall 🤷🏼